lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3388,26 +3388,23 @@ namespace {
                     ->getTopLevelModule()
                     ->getFullModuleName() == n;
         };
-        if ((decl->getOwningModule() &&
-             importer::isCxxStdModule(decl->getOwningModule()))) {
+        clang::Module *owningModule = decl->getOwningModule();
```

Comment:
I think we've bumped to C++17 now. We should be able to fold this into an initializing-if. 
```
if (clang::Module *owningModule = decl->getOwningModule(); owningModule && importer::isCxxStdModule(owningModule)) {
```